src/components/attendee/Input/Input.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { forwardRef } from "react";
+import { FieldError } from "react-hook-form";
+import classNames from "classnames";
+
+import { ContainerClassName } from "types/utility";
+
+import CN from "./Input.module.scss";
+
+interface InputProps
+  extends React.HTMLProps<HTMLInputElement>,
+    ContainerClassName {
+  inputClassName?: string;
+  error?: FieldError;
+  onLabelClick?: () => void;
```

Comment:
[NB] It would be good to add the param and it's type for the event, if such exists for `label`'s `onClick`

Reply:
basically this thing just calls search event ( without any click event info needed ) using form/state value, so I didn't bother to add a type there yet
I guess we can add it in the future if such need arises